garrysmod/lua/postprocess/super_dof.lua

```diff
@@ -120,7 +120,11 @@ function PANEL:Init()
 	local THIS = self
 	function Break:DoClick()
 		THIS:SetVisible( false )
-		timer.Simple( 5, function() THIS:SetVisible( true ) end )
+		timer.Simple( 5, function()
+			if IsValid(THIS) then  

```

Comment:
```suggestion
			if IsValid( THIS ) then  

```
nit: Looks like this file uses spaces inside parens